sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -153,6 +156,56 @@ public partial class SharedSearchServicePrivateLinkResourceData : Azure.Resource
 }
 namespace Azure.ResourceManager.Search.Models
 {
+    public enum AadAuthFailureMode
+    {
+        Http403 = 0,
+        Http401WithBearerChallenge = 1,
+    }
+    public partial class DataPlaneAuthOptions
+    {
+        public DataPlaneAuthOptions() { }
+        public Azure.ResourceManager.Search.Models.AadAuthFailureMode? AadAuthFailureMode { get { throw null; } set { } }
+        public System.BinaryData ApiKeyOnly { get { throw null; } set { } }
+    }
+    public partial class EncryptionWithCmk
+    {
+        public EncryptionWithCmk() { }
+        public Azure.ResourceManager.Search.Models.SearchEncryptionComplianceStatus? EncryptionComplianceStatus { get { throw null; } }
+        public Azure.ResourceManager.Search.Models.SearchEncryptionWithCmk? Enforcement { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PrivateLinkServiceConnectionProvisioningState : System.IEquatable<Azure.ResourceManager.Search.Models.PrivateLinkServiceConnectionProvisioningState>
```

Comment:
Can we rename this enum type to `SearchPrivateLinkServiceConnectionProvisioningState`?